python/pyarrow/parquet/__init__.py

```diff
@@ -3049,6 +3049,11 @@ def file_visitor(written_file):
         # extract non-file format options
         schema = kwargs.pop("schema", None)
         use_threads = kwargs.pop("use_threads", True)
+        chunk_size = kwargs.pop("chunk_size", None)
+        row_group_size = kwargs.pop("row_group_size", None)
+
+        if (row_group_size or chunk_size) is not None:
+            row_group_size = row_group_size if row_group_size else chunk_size
```

Comment:
```suggestion
        row_group_size = row_group_size if row_group_size is not None else chunk_size
```

(slight simplification)